email_account_validity/_servlets.py

```diff
@@ -43,7 +48,8 @@ def __init__(self, config: dict, api: ModuleApi, store: EmailAccountValidityStor
                 "account_renewed.html",
                 "account_previously_renewed.html",
                 "invalid_token.html",
-            ]
+            ],
+            os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates"),
```

Comment:
Will be fixed by matrix-org/synapse#10596, since part of its changeset is adding the new generic custom template directory in the list of directories the module API will look for custom templates in.